core-android/src/main/java/com/uber/sdk/android/core/auth/AuthUtils.java

```diff
@@ -284,6 +286,9 @@ static String buildUrl(
                 .appendQueryParameter(SCOPE_PARAM, getScopes(configuration))
                 .appendQueryParameter(SHOW_FB_PARAM, "false")
                 .appendQueryParameter(SIGNUP_PARAMS, AuthUtils.createEncodedParam(REDIRECT_LOGIN));
+        if (!TextUtils.isEmpty(requestUri)) {
+            builder.appendQueryParameter(REQUEST_URI_PARAM, requestUri);
```

Comment:
why we need to append this in query param? this may contain the user private information that could be logged